ui/app/manager/src/pages/page-configuration.ts

```diff
@@ -0,0 +1,186 @@
+                width: 100%;
+                
+                display: flex;
+                justify-content: center;
+
+                --or-collapisble-panel-background-color: #fff;
+                --or-panel-background-color: #fff;
+                --or-panel-padding: 18px 24px 24px;
+                --or-panel-heading-margin: 0 0 14px 0;
+                --or-panel-background-color: var(--or-app-color1, ${unsafeCSS(DefaultColor1)});
+                --or-panel-heading-font-size: 14px; 
+            }
+
+            or-panel {
+                width: calc(100% - 40px);
+                max-width: 1360px;
+                margin-bottom: 16px;
+            }
+
+            #wrapper {
+                display: flex;
+                min-width: 0px;
+                width: 100%;
+                height: 100%;
+                flex-direction: column;
+                align-items: center;
+                overflow: auto;
+            }
+
+            #header-wrapper {
+                display: flex;
+                width: calc(100% - 40px);
+                max-width: 1360px;
+                padding: 0 20px;
+                flex-direction: row;
+                align-items: center;
+                justify-content: space-between;
+                margin: 15px auto;
+            }
+
+            #header-title {
+                font-size: 18px;
+                font-weight: bold;
+                align-items: center;
+                display: flex;
+                color: var(--or-app-color3, ${unsafeCSS(DefaultColor3)});
+            }
+
+            #header-title > or-icon {
+                margin-right: 10px;
+                margin-left: 14px;
+            }
+
+            #header-actions or-mwc-input{
+                margin-left: 12px;
+            }
+
+            or-icon {
+                vertical-align: middle;
+                --or-icon-width: 20px;
+                --or-icon-height: 20px;
+                margin-right: 2px;
+                margin-left: -5px;
+            }
+
+            @media screen and (max-width: 768px) {
+                or-panel {
+                    border-left: 0px;
+                    border-right: 0px;
+                    width: 100%;
+                    --or-panel-border-radius: 0;
+                }
+                #header-wrapper{
+                    width: calc(100% - 30px);
+                }
+                .hide-mobile {
+                    display: none;
+                }
+            }
+        `;
+    }
+
+    get name(): string {
+        return "configuration";
+    }
+
+    constructor(store: Store<AppStateKeyed>) {
+        super(store);
+    }
+
+    protected render(): TemplateResult | void {
+
+        if (!manager.authenticated) {
+            return html`
+                <or-translate value="notAuthenticated"></or-translate>
+            `;
+        }
+
+
+
+        const app = this
+        document.addEventListener('saveLocalManagerConfig', (e:CustomEvent) => {
```

Comment:
You can safely use `this.addEventListener()` here as far as I'm aware. (same for all other `document` references in the file)

Reply:
After just validation. it won't work when using `this`